Backoffice/AuthorizeEdition/AreaTemplateStrategy.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\AuthorizeEdition;
+
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\TreeNodesPanelStrategy;
+use OpenOrchestra\Backoffice\NavigationPanel\Strategies\TreeTemplatePanelStrategy;
+use OpenOrchestra\ModelInterface\Model\AreaInterface;
+use OpenOrchestra\ModelInterface\Model\TemplateInterface;
+use OpenOrchestra\ModelInterface\Repository\TemplateRepositoryInterface;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+
+/**
+ * Class AreaTemplateStrategy
+ */
+class AreaTemplateStrategy implements AuthorizeEditionInterface
```

Comment:
I strongly disagree with this class.

IMHO, you should do a strategy for the template.

For the area, as the template or the node is giving the status, there should be no strategy on the area